test/invariant/handlers/LpHandler.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { MockERC20 } from "erc20-helpers/MockERC20.sol";
+
+import { HandlerBase } from "test/invariant/handlers/HandlerBase.sol";
+
+import { PSM3 } from "src/PSM3.sol";
+
+contract LpHandler is HandlerBase {
+
+    address[] public lps;
+
+    uint256 public depositCount;
+    uint256 public withdrawCount;
+
+    uint256 public constant TRILLION = 1e10;
+
+    constructor(
+        PSM3      psm_,
+        MockERC20 asset0,
+        MockERC20 asset1,
+        MockERC20 asset2,
+        uint256 lpCount
+    ) HandlerBase(psm_, asset0, asset1, asset2) {
+        for (uint256 i = 0; i < lpCount; i++) {
+            lps.push(makeAddr(string(abi.encodePacked("lp-", i))));
+        }
+    }
+
+    function _getLP(uint256 lpSeed) internal view returns (address) {
+        return lps[_bound(lpSeed, 0, lps.length - 1)];
```

Comment:
It's actually simpler to use mod division here: `return lps[lpSeed % lps.length];`

Reply:
Fixed